test/OpenTelemetry.Instrumentation.Process.Tests/ProcessMetricsTests.cs

```diff
@@ -13,9 +13,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+using System.Collections.Generic;
+using System.Linq;
+using OpenTelemetry.Metrics;
+using Xunit;
 
 namespace OpenTelemetry.Instrumentation.Process.Tests;
 
 public class ProcessMetricsTests
 {
+    private const int MaxTimeToAllowForFlush = 10000;
+
+    [Fact]
+    public void ProcessMetricsAreCaptured()
+    {
+        string metricPrefix = "process.runtime.dotnet.";
+        var exportedItems = new List<Metric>();
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddProcessInstrumentation()
+            .AddInMemoryExporter(exportedItems)
+            .Build();
+
+        meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+
+        Assert.True(exportedItems.Count == 2);
+        var physicalMemoryMetric = exportedItems.FirstOrDefault(i => i.Name == $"{metricPrefix}memory.usage");
```

Comment:
I guess the goal is to test the exact name? Consider using string literal `"process.memory.usage"`.